agent-connection/src/test/kotlin/com/luxoft/blockchainlab/corda/hyperledger/indy/PythonRefAgentConnectionTest.kt

```diff
@@ -52,15 +58,22 @@ class PythonRefAgentConnectionTest {
 
         fun start() {
             val rand = Random().nextInt()
+            val tailsDir = Paths.get(System.getProperty("user.home"),"tails").toFile()
+            if (!tailsDir.exists())
+                tailsDir.mkdirs()
             agentInitEndpoint(agentUrl)
             PythonRefAgentConnection().apply {
                 connect(agentUrl, "User$rand", "pass$rand").toBlocking().value()
                 val invitedPartiesCompleted = mutableListOf<Single<Boolean>>()
                 invitedPartyAgents.forEach { agentUrl ->
                     val party = InvitedPartyProcess(agentUrl)
+                    Paths.get(System.getProperty("user.home"), "tails", party.tailsHash).toFile().writeText(party.tailsHash)
```

Comment:
Lets not make any files outside of project path. And we should clean up after tests.

Reply:
done.